framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -83,10 +84,18 @@ SideSetsAroundSubdomain::modify()
   // Get a reference to our BoundaryInfo object for later use
   BoundaryInfo & boundary_info = mesh.get_boundary_info();
 
+  // Prepare to query about sides adjacent to remote elements if we're
+  // on a distributed mesh
+  const processor_id_type my_n_proc  = mesh.n_processors();
+  const processor_id_type my_proc_id = mesh.processor_id();
+  typedef std::vector<std::pair<dof_id_type, unsigned int> > vec_type;
+  std::vector<vec_type> queries(my_n_proc);
+
   // Loop over the elements
-  MeshBase::const_element_iterator   el  = mesh.active_elements_begin();
-  const MeshBase::const_element_iterator end_el = mesh.active_elements_end();
-  for (; el != end_el ; ++el)
+  for (MeshBase::const_element_iterator
+         el  = mesh.active_elements_begin(),
+         end_el = mesh.active_elements_end();
```

Comment:
They definitely make sense, and we could actually add them before the next release; we'll just have to wait for the first C++11-required libMesh to use them *internally*.